scanner/cmd/updater/main.go

```diff
@@ -4,23 +4,51 @@ import (
 	"context"
 	"flag"
 	"log"
+	"time"
+
+	"errors"
```

Comment:
This needs to go in the block above ^ https://github.com/stackrox/stackrox/actions/runs/6867392818/job/18675612769?pr=8213